logstash.go

```diff
@@ -59,12 +38,33 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 	}, nil
 }
 
+// Returns a new slice containing all strings in the slice that satisfy the predicate f.
+func Filter(vs []string, f func(string) bool) []string {
+	vsf := make([]string, 0)
+	for _, v := range vs {
+		if f(v) {
+			vsf = append(vsf, v)
+		}
+	}
+	return vsf
+}
+
+// Get container tags configured with the environment variable LOGSTASH_TAGS
+func GetContainerTags(containerEnv []string) []string {
```

Comment:
I would prefer it without the Filter function and just have a simple loop here that collects the tags. It just adds complexity and runtime overhead in this case.